actions/milab-shell-utils/bin/gcloud-kms-sign

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+
+: "${GCP_KMS_KEY_VERSION:="1"}"
+: "${GCP_KMS_KEY_NAME:=""}"
+: "${GCP_KMS_KEYRING}:=""}"
+: "${GCP_KMS_LOCATION:="europe-west3"}"
+: "${GCP_KMS_DIGEST_ALGORITHM:="sha256"}"
+
+file_path="${1:-"${FILE_PATH}"}"
```

Comment:
thinking a bit more on it: are `FILE_PATH` and `FILE_SIG_PATH` already provided by some processes?
The names are too general and it may become tricky if we face 'conflict' with som other utilities.

If there is notging to rewrite except shis script - let's name them with `GCP_` prefix, as other variables.

What do you think?

Reply:
Good point, didn't think of it.